games-arcade/rocksndiamonds/rocksndiamonds-4.0.0.2.recipe

```diff
@@ -71,5 +71,7 @@ INSTALL()
 	addResourcesToBinaries rocksndiamonds.rdef \
 		$destdir/rocksndiamonds
 
+	mv $destdir/rocksndiamonds $destdir/"Rocks'n'Diamonds"
+
 	addAppDeskbarSymlink $destdir/rocksndiamonds "Rocks'n'Diamonds"
```

Comment:
Fixed that one, I wanted to use "Rocks'n'Diamonds" also as a binary, but you can't launch it from a Terminal without dropping it into it, so reverted it to "RocksnDiamonds"